ParkingLotTest/ParkingBoyModelsTest.cs

```diff
@@ -0,0 +1,102 @@
+using ParkingLot;
+using ParkingLot.DataModels;
+using ParkingLotTest.Utils;
+using Xunit;
+
+namespace ParkingLotTest
+{
+    public class ParkingBoyModelsTest
+    {
```

Comment:
just suggestion from my opinion, ParkingBoyModels is a little bit hard to understand